credentials-operator/templates/credentials-operator-webhook-service.yaml

```diff
@@ -23,4 +23,24 @@ spec:
     targetPort: 9443
   selector:
     app: credentials-operator
+---
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
+metadata:
```

Comment:
we should probably include the common labels & annotations that are (supposedly) set on all the resources created by our helm charts, i.e. something like: 
```
  labels:
    {{- with .Values.global.commonLabels }}
    {{- toYaml . | nindent 4 }}
    {{- end }}
    app.kubernetes.io/version: {{ .Chart.Version }}
  annotations:
    {{- with .Values.global.commonAnnotations }}
    {{- toYaml . | nindent 4 }}
    {{- end }}
    app.kubernetes.io/version: {{ .Chart.Version }}
```